Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -498,8 +498,9 @@ def setup__(self, step, stepName, stepDict, k, properties):
         if 'Reco' in step: stepDict[stepName][k] = merge([self.step3, stepDict[step][k]])
         elif 'HARVEST' in step: stepDict[stepName][k] = merge([{'-s': 'HARVESTING:@trackingOnlyValidation+@trackingOnlyDQM'}, stepDict[step][k]])
         elif 'ALCA' in step: stepDict[stepName][k] = None
-    def condition_(self, fragment, stepList, key, hasHarvest):
-        return ('Run4' in key)
+    def condition(self, fragment, stepList, key, hasHarvest):
+        result = (fragment=="TTbar_14TeV") and hasHarvest and ('Run4' in key)
```

Comment:
Just for my curiosity (already signed): why limiting this to `TTbar`? Could be useful to have it defined for some displaced topologies?

Reply:
The workflow was built to inherit from `UpgradeWorkflowTracking`, so what I did here was to override this line:
https://github.com/cms-sw/cmssw/blob/6c309578b4970a207343f7ae56a6decd242bdb8e/Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py#L298
to include the one sample of Phase 2 and to remove `not PU` restriction.
I agree that more workflows would be useful but there will be more work on our workflows in the future, which will address this point as well.